src/Symfony/Component/Cache/DependencyInjection/CacheCollectorPass.php

```diff
@@ -26,31 +26,18 @@
  */
 class CacheCollectorPass implements CompilerPassInterface
 {
-    private $dataCollectorCacheId;
-    private $cachePoolTag;
-    private $cachePoolRecorderInnerSuffix;
-
-    public function __construct(string $dataCollectorCacheId = 'data_collector.cache', string $cachePoolTag = 'cache.pool', string $cachePoolRecorderInnerSuffix = '.recorder_inner')
-    {
-        if (0 < \func_num_args()) {
-            trigger_deprecation('symfony/cache', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
-        }
-
-        $this->dataCollectorCacheId = $dataCollectorCacheId;
-        $this->cachePoolTag = $cachePoolTag;
-        $this->cachePoolRecorderInnerSuffix = $cachePoolRecorderInnerSuffix;
-    }
+    private const DATA_COLLECTOR_CACHE_ID = 'data_collector.cache';
```

Comment:
I'd personally prefer having no consts for these

Reply:
Also for values which are 4 times in a class?

Reply:
yes...

Reply:
Okay, no problem :)